airflow/www/views.py

```diff
@@ -1084,6 +1084,28 @@ def blocked(self):
             })
         return wwwutils.json_response(payload)
 
+    @expose('/latest_dag_runs')
+    @login_required
+    def latest_dag_runs(self):
```

Comment:
@bolkedebruin is the business logic okay as a class method in models.Dag or models.DagRun or are we moving in the direction of putting things like this into API